### PR TITLE
Phonebook

### PR DESCRIPTION
**Pull request by arvinth10**
- Add sms option
- Add calling option

### CODE REVIEW

File: contacts_app/lib/helperFunctions.dart [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            .catchError((dynamic onError) {
+      print(onError);
+      standardAlertDialog(context, "Error", onError.toString());
+    });
+    print(_result);
+  }
+
+  static Future<void> callNumber(BuildContext context, String number) async {
+    bool _result =
+        await launch('tell://' + number).catchError((dynamic onError) {
```
Comment (JasdipChauhan) [inline contacts_app/lib/helperFunctions.dart]: isn't it usually `tel`?

File: contacts_app/lib/helperFunctions.dart [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      print(onError);
+      standardAlertDialog(context, "Error", onError.toString());
+    });
+    if (_result == false) {
+      standardAlertDialog(context, "Error", "Can not call on this device");
+    }
+  }
+
+  static String hasValidPhoneNumber(Iterable phoneNumbers) {
```
Comment (JasdipChauhan) [inline contacts_app/lib/helperFunctions.dart]: nit pick but hasValidPhoneNumber sounds like it should return a boolean

File: contacts_app/lib/helperFunctions.dart [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+import 'package:url_launcher/url_launcher.dart';
+import 'messagingDialog.dart';
+
+class HelperFunctions {
+  static Future<void> initiateMessage(
+      BuildContext context, String message, String recipient) async {
+    final String _result =
+        await sendSMS(message: message, recipients: <String>[recipient])
+            .catchError((dynamic onError) {
+      print(onError);
```
Comment (yenvanio) [inline contacts_app/lib/helperFunctions.dart]: Should the final code have print statements?

File: contacts_app/lib/contactsPage.dart [hunk 4]
```diff
@@ -1,5 +1,7 @@
 import 'package:flutter/material.dart';
 import 'package:contacts_service/contacts_service.dart';
+import 'smsButton.dart';
```
Comment (yenvanio) [inline contacts_app/lib/contactsPage.dart]: Might add style points if you do import 'package:<package_name>/smsButton'

**Review by yenvanio — COMMENTED** [review]
code lgtm for the most part, left some small comments

File: contacts_app/lib/helperFunctions.dart (hunk 3 above)
Comment (arvinth10) [reply contacts_app/lib/helperFunctions.dart]: Thought it would be a good log to keep? I could remove it though if you think we shouldnt have it

**Review by yenvanio — APPROVED** [review]

**Pull request merged**